test/unit/phys/parameters/four_point_parameters/four_point_parameters_test.cpp

```diff
@@ -40,7 +41,8 @@ TEST(FourPointParametersTest, ReadAll) {
   EXPECT_EQ(dca::phys::PARTICLE_PARTICLE_UP_DOWN, pars.get_four_point_type());
   EXPECT_EQ(momentum_transfer_input_check, pars.get_four_point_momentum_transfer_input());
   EXPECT_EQ(1, pars.get_four_point_frequency_transfer());
-  
+  EXPECT_EQ(1, pars.compute_all_transfers());
```

Comment:
/pedantic Should be 'true' rather than 1, since it's a bool and you used 'false' for the other test

Reply:
ofc, changed in 3f740b8f71284b0ee39e088eb0d6962cecd47f0f.